contracts/src/v0.8/ccip/test/pools/GHO/fork/GhoTokenPoolMigrate1_4To1_5/LegacyTestnetTokenPools/UpgradeableBurnMintTokenPool_ArbSepolia.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {ITypeAndVersion} from "../../../../../../../shared/interfaces/ITypeAndVersion.sol";
+import {IBurnMintERC20} from "../../../../../../../shared/token/ERC20/IBurnMintERC20.sol";
+import {UpgradeableTokenPool} from "../../../../../../pools/GHO/UpgradeableTokenPool.sol";
+import {UpgradeableBurnMintTokenPoolAbstract} from "../../../../../../pools/GHO/UpgradeableBurnMintTokenPoolAbstract.sol";
+import {IRouter} from "../../../../../../interfaces/IRouter.sol";
+import {VersionedInitializable} from "./VersionedInitializable.sol";
+
+/// @title UpgradeableBurnMintTokenPool_ArbSepolia
+/// @author Aave Labs
+/// @notice Upgradeable version of Chainlink's CCIP BurnMintTokenPool
+/// @dev Contract adaptations:
+/// - Implementation of VersionedInitializable to allow upgrades
+/// - Move of allowlist and router definition to initialization stage
+contract UpgradeableBurnMintTokenPool_ArbSepolia is
+  VersionedInitializable,
+  UpgradeableBurnMintTokenPoolAbstract,
+  ITypeAndVersion
+{
+  string public constant override typeAndVersion = "BurnMintTokenPool 1.4.0";
+
+  /// @dev Constructor
+  /// @param token The bridgeable token that is managed by this pool.
+  /// @param armProxy The address of the arm proxy
+  /// @param allowlistEnabled True if pool is set to access-controlled mode, false otherwise
+  constructor(
+    address token,
+    address armProxy,
+    bool allowlistEnabled
+  ) UpgradeableTokenPool(IBurnMintERC20(token), armProxy, allowlistEnabled) {}
+
+  /// @dev Initializer
+  /// @dev The address passed as `owner` must accept ownership after initialization.
+  /// @dev The `allowlist` is only effective if pool is set to access-controlled mode
+  /// @param owner The address of the owner
+  /// @param allowlist A set of addresses allowed to trigger lockOrBurn as original senders
+  /// @param router The address of the router
+  function initialize(address owner, address[] memory allowlist, address router) public virtual initializer {
+    if (owner == address(0)) revert ZeroAddressNotAllowed();
+    if (router == address(0)) revert ZeroAddressNotAllowed();
```

Comment:
Similarly to above, can make this 1 combined if statement

Reply:
same as https://github.com/aave/ccip/pull/18#discussion_r1803241138